### PR TITLE
Add print_operand function and clean up related code

### PR DESCRIPTION
**Pull request by aemmitt-ns**
This change adds the print_operand function to better organize the code and standardize the way operand values are added to the ESIL string. Previously there were issues where some operations with operands that were memory dereferences were not written properly so they would just use the address as the value. 

Additionally this removes the ",NUM" expressions that were not necessary here, and changes "==" to "-,!" since that was the desired behavior. In general there is less copy and pasted code which should be good for making changes in the future.

This was tested mainly by analyzing functions after running `pdga` and emulating with ESIL. Results were as expected.

### CODE REVIEW

File: src/anal_ghidra.cpp [hunk 1]
```diff
@@ -823,6 +823,23 @@ static void sleigh_esil(RAnal *a, RAnalOp *anal_op, ut64 addr, const ut8 *data,
...
 	};
 
+	auto print_operand = [&esil_stack, &ss](const PcodeOperand *arg, int offset = 0) -> bool {
+		if(arg->is_unique()) {
+			int index = index_of_unique(esil_stack, arg);
+			if(-1 == index)
+				throw LowlevelError(
+				    "print_if_unique: Can't find required unique varnodes in stack.");
+
+			ss << index + offset << ",PICK";
```
Comment (trufae) [inline src/anal_ghidra.cpp:833]: i think @condret will complain about this PICK :D

**Review by trufae — CHANGES_REQUESTED** [review]

File: src/anal_ghidra.cpp (hunk 1 above)
Comment (aemmitt-ns) [reply src/anal_ghidra.cpp:833]: Yea I am going to try to remove the PICKs. Also there is something still off with the FP ops. Well not the operations but assigning to registers doesn't work properly. gonna convert to draft

Comment (trufae) [reply src/anal_ghidra.cpp:833]: Actually there's no PICK in the esil vm, but it doesn't seems to complain when evaluating expressions with unknown operations
Screenshot 2020-12-22 at 01 57 06

**Comment by trufae** [comment]
Please rebase and fix the conflicts lots of things has changed

**Pull request closed without merging**